README.md

```diff
@@ -315,6 +315,18 @@ setVerbose(True)
 # turn off verbose mode
 setVerbose(False)
 ```
+### Bypassing merge task 
+
+Skipping the merge task is useful when the tasks results don't need to be merge into a list. To bypass the merge task, you can pass the *enableMerge* flag to the foreach object:

```

Comment:
Typo. Make it:

Skipping the merge task is useful when the tasks results don't need to be merged into a list. To bypass the merge task, you can pass the *enableMerge* flag to the foreach object:

Also, should we hyperlink the 'merge task' at the beginning to more info about what the merge task is?